supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -103,6 +114,18 @@
 
 *See also*:
 
+[[thickly-provisioned]]
+==== image:images/yes.png[yes] thickly provisioned (adjective)
+*Description*: Disk storage that is pre-allocated so that the total amount of virtual disk storage capacity exists on the physical storage when a virtual disk is created.
```

Comment:
```suggestion
*Description*: _Thickly provisioned_ is used with reference to disk storage that is pre-allocated so that the total amount of virtual disk storage capacity exists on the physical storage when a virtual disk is created.
```